src/style.css

```diff
@@ -0,0 +1,65 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style-type: none;
+}
+
+body {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  padding: 1em;
+}
+
+.container {
+  max-width: 50%;
+  border: 2px solid green;
+  margin: 1em auto;
+  padding: 3em;
+}
+
+.title {
+  margin-bottom: 2em;
+}
+
+.details {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```

Comment:
- [ ] Please fix the below style issues to match the wireframe design.
  - Please make sure the refresh button is aligned at the end of the score section. Currently, it has some margin from the right.
  - Please make sure the submit button is aligned to the right instead of the left.


### Your design

![image](https://github.com/rabayarabu/Leaderboard/assets/83803414/0980332c-cb7e-4ee2-a9f8-3623e1a0cb7c)

### Wireframe design

![image](https://github.com/rabayarabu/Leaderboard/assets/83803414/f4954df6-0b5b-4afb-9bce-57b3595c8c97)